notebooks/featurestore/aws/redshift/Redshift_pyspark.ipynb

```diff
@@ -20,7 +20,7 @@
     "\n",
     "This notebooks guides through the ingestion of Redshift data in the Hopsworks feature store. To follow this notebook users should have an existing Redshift cluster, if not, they can follow the AWS [documentation](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/event-publishing-redshift-cluster.html).\n",
     "\n",
-    "The data for this tutorial is available in CSV format [here](../data/Sacramentorealestatetransactions.csv)\n",
+    "We use the widely used Telco Customer Churn dataset for this tutorial, which is available in CSV format [here](../data/Telco_customer_churn.csv)\n",
```

Comment:
can we put this dataset to https://repo.hops.works/matser or link to any public ones?

Reply:
I downloaded this from kaggle https://www.kaggle.com/datasets/blastchar/telco-customer-churn 
Should we point to this instead of local links?

Reply:
lets upload this to here https://repo.hops.works/matser so people can read it directly. Like this `subscriptions_df = pd.read_csv("https://repo.hops.works/dev/davit/churn/subscriptions.csv")`

Reply:
added csv to https://repo.hops.works/master/hopsworks-tutorials/data/Telco_customer_churn.csv